Altis_Life.Altis/core/actions/fn_catchFish.sqf

```diff
@@ -2,29 +2,43 @@
 /*
     File: fn_catchFish.sqf
     Author: Bryan "Tonic" Boardwine
+    Modified and optimised by SimZor
```

Comment:
Leave original author. All edits are logged and can be viewed via [history](https://help.github.com/articles/differences-between-commit-views/).